### DISCUSSION TITLE
[Feature] Grouping media by type or file size

### DISCUSSION THREAD

**Discussion by maffinca69** [immich-app/immich#7281] [Feature Request]
### The feature

Now in the default photo app there is functionality to select only screenshots. This makes it easier to clean the gallery, because once every couple of months you can clear them all out and save space. In immih (if the images are on the server, but not on the phone) there is no such possibility
An alternative would be the functionality of grouping media by file size, so that large files (videos, raw photos, etc.) can be tracked and deleted (perhaps the immich itself will periodically notify you about it)

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
There is now https://my.immich.app/utilities/large-files for this.
